spec/controllers/staff/activity_forms_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe Staff::ActivityFormsController do
```

Comment:
I'm finding these specs tricky to follow. Can we have the expectations inline, rather than using custom matchers? Something like:

```
context "when editing a fund" do
...
  it "skips the parent step" do
    expect { get_step :parent }.to redirect_to(next_step)
  end
...
```

Reply:
This is what I get for trying to adopt nested `let` blocks (which I generally loathe for readability!)

Reply:
@pezholio On second thoughts, I'm wondering if using inline expectations coupled with custom matchers is best here - the important point to these tests is that *either* the current step is rendered *or* we redirect to the next step (whatever that might be), and I'd like to avoid yet another place where the order of the steps matters.

i.e. would prefer `expect { ... }. to redirect_to_next_wizard_step` over `redirect_to(wizard_path(... :identifier))`

Reply:
OK, cool. Let's do that. I think having the custom matcher more visible would help readability too 👍